### ISSUE TITLE
Plungin stopped working

### ISSUE THREAD

**Issue by JD719** [exebetche/vlsub#175]
Hi, I'm aware opensubtitles made changes in their end yesterday, from them vlsub stopped working.
It hangs on "searching subtitles"

Debian 9.0
VLC 2.2.5
VLSub 0.9

This is the log I was able to capture running VLC from CLI

> [00007fbe44001638] http access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe48c9be78] http access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe48c9be78] http access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe44001638] http access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe44001638] access_mms access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe44000958] core art finder error: no suitable access module for `http://services.tvrage.com/feeds/search.php?show=Black Sails'
> [00007fbe48c9be78] access_mms access error: error: HTTP/1.1 522 Origin Connection Time-out
> [00007fbe4800fa28] core art finder error: no suitable access module for `http://services.tvrage.com/feeds/search.php?show=Black Sails'

**Comment by harrierbird** [comment]
yeah, I am having the same problem. Search results = 0. Either that or it crashes.

**Comment by varjolintu** [comment]
Version 0.9? Why don't you try if it happens with the latest version?

**Comment by JD719** [comment]
@varjolintu 0.9 is the latest version available in my distro repo.
Isn't vlsub pushing updates to debian anymore?

**Comment by varjolintu** [comment]
@JD719 I really don't now about Debian repo's. The VLC addon page is out of date, so the repo should be updated from the github. Still, it shows 24 commits after the last release so..

**Comment by JD719** [comment]
ok I just downloaded the last available version, now I get the same result as @harrierbird 
0 results

**Comment by varjolintu** [comment]
What changes did opensubtitles made? I couldn't find them. I tested the plugin today and it worked normally.

**Comment by JD719** [comment]
It's not clear to me, their explanations are pretty vague
You can get some info here: http://forum.opensubtitles.org/viewforum.php?f=1

**Comment by harrierbird** [comment]
Thank you @JD719. I was staring to think I was the only one having this problem with the new VLSub version. I use what I think is the latest one, v0.10.2.

**Comment by JD719** [comment]
I should specify something, the feature that is not working is the "search by hash" one, the one I always used. Search by name seems to work, but it is way less reliable in my experience.

**Comment by varjolintu** [comment]
The reason the plugin works for me is probably that the metadata scripts are not ran at all:

> core debug: looking for meta fetcher module matching "any": 1 candidates
> lua debug: Trying Lua scripts in /Users/humanoid/Library/Application Support/org.videolan.vlc/lua/meta/fetcher
> lua debug: Trying Lua scripts in /Applications/VLC.app/Contents/MacOS/share/lua/meta/fetcher
> lua debug: Trying Lua playlist script /Applications/VLC.app/Contents/MacOS/share/lua/meta/fetcher/tvrage.luac
> lua debug: skipping script (unmatched scope) /Applications/VLC.app/Contents/MacOS/share/lua/meta/fetcher/tvrage.luac
> lua debug: Trying Lua scripts in /Applications/VLC.app/Contents/MacOS/share/share/lua/meta/fetcher
> core debug: no meta fetcher modules matched

Is there any difference if you open the VLC Preferences and uncheck the checkbox which reads "Allow metadata network access"?

**Comment by harrierbird** [comment]
I use "Search by name" and mysteriously, this now works again... O.o

**Comment by JD719** [comment]
@varjolintu I'm afraid it makes no difference for me.
@harrierbird yes as I stated before it works for me too, but searching by name is kinda sad, the chances you get a synchronized sub file decrease drastically.

**Comment by varjolintu** [comment]
For me it seems that the OpenSubtitles is limiting the API requests for some reason.

@JD719, have you tried to remove the tvrage.luac script from VLC? Does it make a difference?

**Comment by JD719** [comment]
@varjolintu it does not make difference

**Comment by marsjaninzmarsa** [comment]
Are you sure that you're using newest version of VLSub? VLC can load extensions from many locations, and sometimes more than one version of extension can be loaded at the same time.
![zrzut ekranu z 2017-04-13 13 20 58](https://cloud.githubusercontent.com/assets/1119575/25002731/324ab4ce-204c-11e7-95f0-236185f76721.png)

**Comment by JD719** [comment]
@marsjaninzmarsa yes I'm using 0.10.2. I actually deleted the old one.
Do you get results searching by hash?

**Comment by harrierbird** [comment]
@JD719 I am using 0.10.2 and I get results searching by hash, even if it's just one. What happened to me seemed to be something temporary. It eventually went away without doing anything. Not sure what is happening to you though.

**Comment by JD719** [comment]
@harrierbird what OS are you running VLC on?

**Comment by harrierbird** [comment]
@JD719 macOS Sierra 10.12.14

**Comment by pdrocaldeira** [comment]
Not working for me, using 0.10.2. :(

Arch Linux .

**Comment by marsjaninzmarsa** [comment]
@JD719 worksforme, VLSub 0.10.2 and 0.10.1, VLC 2.2.4, LMDE2 and Windows 7 (3 machines total).

Are you logged in? OS was known for limiting unauthenticated API calls.

**Comment by JD719** [comment]
After a few days downloading subs manually, today I did a search within the plugin and got one result at least, I used to get 3 at least for series like this one. It must be something in OS end they screwed.

@marsjaninzmarsa no I'm not using credentials, never did it, anyway if I log in the result is the same.
